rotary_jet_spinning/datas/data.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Source : A simple model for nanofiber formation by rotary jet-spinning
```

Comment:
We could include a .bibfile in the repo (it would be easy as we already have the library on Zotero, we just need to export it as a bib file) and use the unique identifiers for each publication in the python code (such as [mellado2011]). It makes it easier to explore the code and "insert citation" within it. If someone wants the details of the paper, they can just open the bibfile in the repo and access them.